rs/nns/governance/src/governance.rs

```diff
@@ -2391,22 +2403,20 @@ impl Governance {
             return Err(GovernanceError::new(ErrorType::NotAuthorized));
         }
 
-        if split_amount_e8s < min_stake + transaction_fee_e8s {
+        let min_split_amount = min_stake.saturating_add(transaction_fee_e8s);
```

Comment:
Here, I think we can say that we know that the two operands to saturating_add are small, and thus, there would be no overflow, and so it is ok to use saturating_add.